docs/astro/src/content/docs/reference/common.mdx

```diff
@@ -49,39 +49,57 @@ the element and its children with transparency.
 The opacity is applied to the tree of child elements as if they
 were first drawn into an intermediate layer, and then the whole layer is rendered with this opacity.
 
-:::tip[Tip]
-When an element has 0 opacity it will still take up layout space and any gesture handling will continue
-to work. If the intent is to hide an element so it has no gesture handling or no longer takes up layout space,
-use the `visible` property instead.
-:::
+<CodeSnippetMD imagePath="/src/assets/generated/rectangle-opacity.png" scale="3" imageWidth="100" imageHeight="310"  imageAlt='rectangle opacity'>
+```slint playground
+component ImageInfo inherits Rectangle {
```

Comment:
This example is fine. But it deosn't explain the layer approach.
Which mean that for example, if the Rectangle itself has an opacity, both we can see through both the image and the text, but we can't see the image through the text.

See the Qt documentation as a better explaination for what i'm trying to explain: https://doc.qt.io/qt-6/qml-qtquick-item.html#item-layers

We pretend to operate with a "Layered Opacity"

Unless we are using the software renderer, in which case this is currently implemented with a "Non-layered Opacity" which i consider a defect of the software renderer, but i don't know if we can/want to fix that without paying a price of quite a lot of memory